core/dbt/parser/manifest.py

```diff
@@ -493,7 +499,10 @@ def is_partial_parsable(self, manifest: Manifest) -> Tuple[bool, Optional[str]]:
         reparse_reason = None
 
         if manifest.metadata.dbt_version != __version__:
-            logger.info("Unable to do partial parsing because of a dbt version mismatch")
+            # #3757 log both versions because of reports of invalid cases of mismatch.
+            logger.info("Unable to do partial parsing because of a dbt version mismatch. "
+                        "Saved manifest version: {manifest.metadata.dbt_version}. "
+                        "Current version: {__version__}.")
```

Comment:
Does this work in Python without the `f` for inserting variables? I honestly don't know so asking for myself

Reply:
Ya, good call!

Reply:
Thanks!